qa/common/infer_util.py

```diff
@@ -137,21 +146,20 @@ def infer_exact(tester, pf, tensor_shape, batch_size,
                          for x in input1_array.reshape(input1_array.size)], dtype=object)
         input1_array = in1n.reshape(input1_array.shape)
 
-    # prepend size of string to input string data
-    if input_dtype == np.object:
-        input0_array_tmp = serialize_byte_tensor(input0_array)
-        input1_array_tmp = serialize_byte_tensor(input1_array)
-    else:
-        input0_array_tmp = input0_array
-        input1_array_tmp = input1_array
-
+    # prepend size of string to output string data
     if output0_dtype == np.object:
-        output0_array_tmp = serialize_byte_tensor(output0_array)
+        if batch_size == 1:
+            output0_array_tmp = _prepend_string_size([output0_array])
```

Comment:
This is needed just to get the output_byte_size right? Why not get the size directly without serializing first?

Reply:
Sure, I could do that